### ISSUE TITLE
lazygit's content is not visible with a light theme

### ISSUE THREAD

**Issue by devcui** [LunarVim/LunarVim#2165]
### Problem description

There has Some contents are not clear in lunarvim's lazy git.

### LunarVim version

d084051

### Neovim version (>= 0.6)

v0.6.1

### Operating system/version

macOS 12.0.1

### Relevant log output

```shell
none
```

### Screenshots

![image](https://user-images.githubusercontent.com/26159886/148893775-68548cb5-2fb5-4428-9c00-7ad2c2dcdb5a.png)

and when i enter Ctrl+a

![image](https://user-images.githubusercontent.com/26159886/148893796-0fc7755a-1073-498a-888a-8a1a1a90b7e5.png)

that's my config https://github.com/devcui/lunarvim-config/blob/main/config.lua 
theme edge : https://github.com/sainnhe/edge

**Comment by kylo252** [comment]
This is generally solved from lazygit's side but maybe we could put a note about that in the docs. 

tl;dr; 
1. run this in your terminal
```console
# it should be your XDG_CONFIG_HOME
lazygit --config > ~/Library/Application Support/lazygit/config.yml
```
2. change the `theme` entry to your heart's content

You can also use a different config file per invocation.

reference: https://github.com/jesseduffield/lazygit/blob/4744b39f038d9e197455dc81a55277e1b1fe0586/docs/Config.md

**Comment by devcui** [comment]
@kylo252 thanks, and i have a new question, I don't know how to globally search a file that contains text,Maybe I should install a plugin?

**Comment by kylo252** [comment]
> @kylo252 thanks, 

You're welcome!

> and i have a new question, I don't know how to globally search a file that contains text(not leader f,the text in the file)

The one you want it `<leader>st`, the mnemonic being "search text".

We use https://github.com/nvim-telescope/telescope.nvim which already offers a lot of commands, including the one you're looking for `:Telescope live_grep`. A lot of telescope's search commands are bound to `<leader>s`. Take a look here for the full list: https://github.com/LunarVim/LunarVim/blob/rolling/lua/lvim/core/which-key.lua#L61-L246

**Comment by devcui** [comment]
@kylo252  Great. That's what I'm looking for. Thank you very much.😊

**Comment by devcui** [comment]
to resolve the first question:
change lazygit's config, open the `lightTheme`

```
  theme:
    lightTheme: true

```
and change the `inactiveBorderColor`
```
    inactiveBorderColor:
    - black
```

---

resolve the second question:

```
brew install ripgrep
```

@kylo252 thanks again🙏
